dftimewolf/lib/timesketch_utils.py

```diff
@@ -9,7 +9,7 @@
 LOCK = threading.Lock()
 
 
-def GetApiClient(state):
+def GetApiClient(state, token_password=''):
```

Comment:
Shouldn't this default to `None`? It feels like encrypting with an empty password should not be the same as not encrypting the file. (unless you're XORing 😅)

Reply:
well, no, the password is a string, and the upstream library expects a string to be passed in, and if the string is empty then the `default` pwd is used, which is auto generated

Reply:
Ack, sounds good. Could you add a bit of docstring that says this?

Reply:
sure can....